scripts/form.js

```diff
@@ -1,21 +1,41 @@
 const emailInput = document.querySelector('#user-email');
 const form = document.querySelector('#signup');
 
-const showErrorMsg = (input, message) => {
-  const error = document.querySelector('small');
+const destopForm = document.querySelector('#desktopSignup');
+const userEmail = document.querySelector('#useremail');
+
+const showErrorMsg = (type) => {
+  const error = type === 'mobile' ? document.querySelectorAll('small')[0] : document.querySelectorAll('small')[1];
   error.style.color = 'red';
   error.marginTop = '8px';
-  error.textContent = message;
+  error.textContent = 'email address should be in lowercase';
 };
 
-form.addEventListener('submit', (e) => {
-  e.preventDefault();
+const showSuccessMsg = () => {
+  const sucess = document.querySelector('small');
+  sucess.style.color = 'green';
+  sucess.marginTop = '8px';
+  sucess.textContent = 'Thank you!';
+};
+
+destopForm.addEventListener('submit', (e) => {
+  const email = userEmail.value.trim();
+  const convertedEmail = email.toUpperCase();
+  if (email === convertedEmail) {
+    showErrorMsg('desktop');
+    e.preventDefault();
+  } else {
+    showSuccessMsg();
+  }
+});
```

Comment:
When I submit the form with the email field containing uppercase characters nothing happens. Kindly look into this and make sure that:

- The error message should appear close to the "submit" button.
- The form should not be sent even if there's only one uppercase character on the email address. Currently, when I submit the form with the email having the first character as uppercase e.g `Example@email.com` the form is still submitted. This should not happen.

Why don't you try to check if the email is all lowercase by comparing if the input email and its lowercase converted version are the same instead:
```
email === email.toLowerCase()
```

Reply:
@KDlamini that was a logical error right there. `.toLowerCase()` does a strict check. thank you.